### ISSUE TITLE
Typos / errors in instructions for building ISO

### ISSUE THREAD

**Issue by izogfif** [ionuttbara/windows-defender-remover#152]
The README.md file in the root of this repository has formatting issues that make instructions hard to follow (at least when viewed via GitHub HTML viewer for .md files): "dollar-OEM-dollar" is displayed as some italicized "OEM". Later, the "slash" character is not displayed properly (probably because Markdown processor treats this slash character as a part of an escape sequence).

Also, instructions mention "unnatended.xml", but there is no such file in this repository. There is "unnatend.xml" instead.

And this "unattend.xml" file doesn't look like a valid XML file: there is an error at line 116: there is a closing `</component>` tag without closing `</RunSynchronous>` tag.

**Comment by ionuttbara** [comment]
ok. thanks for noticing. i testing in Window 11 only. 
always i download source code first before verification. i will get an eye on that error.

**Comment by ionuttbara** [comment]
you must put the $OEM$ in sources folder of the iso and you're finished. You can do that with MagicISO fast. (no sponsored). 
note: this works on all Windows version except Windows 11 24H2 (24H2 enabled in tamper protection in pre-OOBE phase and i don't know how to disable from that for now.)

**Comment by ionuttbara** [comment]
will be corrected. found some methods, but it will be closed due long time.
